SharpBot.Data/GuildService.cs

```
@@ -38,5 +38,16 @@ public void SetPrefix(ulong guildId, char prefix)
             connection.Open();
             connection.Execute($"select set_prefix('{guildId}', '{prefix}')");
         }
+
+        public void SetMessageLog(ulong guildId, bool value)
+        {
+            using var connection = new NpgsqlConnection(connectionString);
+            connection.Open();
+            connection.Execute($"select set_message_log('{guildId}', {value})");
+
+            if (value)
+                try { connection.Execute($"select create_message_partition('{guildId}')"); }
+                catch { }
```

I know we talked about empty catches and fixing em later, I'm just pointing it out again so it doesn't drop off our radar.

---

yep, i put it on trello